app/client/packages/design-system/widgets/src/components/Avatar/src/Avatar.tsx

```
@@ -0,0 +1,37 @@
+    }
+
+    return `${names[0].charAt(0)}${names[1]?.charAt(0)}`;
+  };
+
+  return (
+    <span
+      className={clsx(styles.avatar, className)}
+      {...rest}
+      data-size={size ? size : undefined}
+    >
+      {Boolean(src) ? (
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)